src/ops/dataflow.rs

```diff
@@ -161,6 +182,21 @@ impl Call {
     }
 
     /// The IncomingPort which links to the function being called.
+    ///
+    /// This matches [`OpType::static_input_port`].
+    ///
+    /// ```
+    /// # use hugr::ops::dataflow::Call;
+    /// # use hugr::ops::OpType;
+    /// # use hugr::types::FunctionType;
+    /// # use hugr::extension::prelude::QB_T;
+    /// let signature = FunctionType::new(vec![QB_T, QB_T], vec![QB_T, QB_T]);
+    /// let call = Call { signature };
+    /// let op = OpType::Call(call.clone());
+    /// assert_eq!(op.static_input_port(), Some(call.called_function_port()));
+    /// ```
+    ///
+    /// [`OpType::static_input_port`]: crate::ops::OpType::static_input_port
     #[inline]
     pub fn called_function_port(&self) -> IncomingPort {
```

Comment:
do we still need this method (or the matching one for Const for that matter)?

Reply:
They are needed if we use the specific `Call`/`LoadConstant` ops directly, as `static_port` is only implemented for `OpType`.

We _could_ add that method to `DataflowOpTrait` too?
Though here I'd say that your comment applies; in that if you're at this level you are dealing with specific operations.
(Alternatively we can always `Into<OpType>` and call those methods)